### ISSUE TITLE
Add requirements.txt and specify minimum version of ducktape

### ISSUE THREAD

**Issue by ewencp** [confluentinc/muckrake#15]
Right now we're making a bunch of changes in ducktape and muckrake in parallel, but once ducktape settles down a bit, we'll need to use a requirements.txt file to pin to a minimum version of ducktape (which presumably will be released in PyPI) so that automated tests can run reliably. (This also means we'll need to be more disciplined about developing features in ducktape, releasing, and only then updating muckrake to use them.)

**Comment by ewencp** [comment]
Duplicate of #2
